website/docs/for-android/how-to/using-a-capacitor-plugin.md

```diff
@@ -310,36 +170,18 @@ For more information on how to use Capacitor Plugins in your web application, [c
 
 Any plugin built for Capacitor can be linked in to a project using Portals even if it is not available through public native repositories like the core plugins are.
 
-### Download the Plugin
+#### Download the Plugin
```

Comment:
I think an extra title level was added here by mistake. The `Other Plugins` section is ## but then this and others below jump to ####.